get_all_manifests.sh

```
@@ -57,26 +57,52 @@ declare -A RHOAI_COMPONENT_MANIFESTS=(
 
-# {ODH,RHOAI}_COMPONENT_CHARTS are lists of chart repositories info to fetch helm charts
+# {ODH,RHOAI}_{CCM,COMPONENT}_CHARTS are lists of chart repositories info to fetch helm charts
 # in the same format as manifests: "repo-org:repo-name:ref-name:source-folder"
 # key is the target folder under charts/
+# CCM_CHARTS: charts deployed by the CloudManager controller (dependencies)
+# COMPONENT_CHARTS: charts deployed by individual component controllers
 
-# ODH Component Charts
-declare -A ODH_COMPONENT_CHARTS=(
+# ODH CloudManager Charts
+declare -A ODH_CCM_CHARTS=(
```

Shellcheck (0.11.0)

[warning] ODH_CCM_CHARTS appears unused. Verify use (or export if used externally).

(SC2034)